### PR TITLE
SA22-029 Rename also in comments

### PR DESCRIPTION
**Pull request by ogorodnik**
Added:
  - renameInComments configuration option to control whether
       process also comments when renaming or not
  - process comments and includes locations in response when
       comment should be changed

### CODE REVIEW

File: source/ada/lsp-ada_handlers.adb [hunk 1]
```diff
@@ -1517,6 +1518,43 @@ package body LSP.Ada_Handlers is
          Definition : Defining_Name;
          Imprecise  : Boolean;
          Empty      : LSP.Messages.TextEdit_Vector;
+
+         procedure Process_Comments
+           (Node : Ada_Node;
+            Uri  : LSP.Messages.DocumentUri);
```
Comment (setton) [inline source/ada/lsp-ada_handlers.adb]: needs doc

File: source/ada/lsp-lal_utils.ads [hunk 2]
```diff
@@ -90,4 +98,11 @@ package LSP.Lal_Utils is
    --  Imprecise_Results is set to True if we don't know whether the results
    --  are precise.
 
+   function Contain
```
Comment (setton) [inline source/ada/lsp-lal_utils.ads]: could this be renamed to `Contains` ?

File: source/ada/lsp-lal_utils.ads [hunk 3]
```diff
@@ -41,6 +44,11 @@ package LSP.Lal_Utils is
    --  Imprecise is set to True if LAL's imprecise fallback mechanism has been
    --  used to compute the cross reference.
 
+   function Get_Last_Name
```
Comment (setton) [inline source/ada/lsp-lal_utils.ads]: (style) join this line with the next line (same comment in the body)

File: source/ada/lsp-ada_handlers.adb (hunk 1 above)
Comment (ogorodnik) [reply source/ada/lsp-ada_handlers.adb]: done

File: source/ada/lsp-lal_utils.ads (hunk 2 above)
Comment (ogorodnik) [reply source/ada/lsp-lal_utils.ads]: done

File: source/ada/lsp-lal_utils.ads (hunk 3 above)
Comment (ogorodnik) [reply source/ada/lsp-lal_utils.ads]: done

**Comment by ogorodnik** [comment]
New patch has been created

**Pull request closed without merging**